backend/models/User.js

```diff
@@ -28,17 +29,28 @@ const User = new Schema({
 		type: String,
 		trim: true,
 		minlength: [2, 'First names must be at least 2 characters.'],
-		maxlength: [30, 'First names must be no longer than 30 characters.']
+		maxlength: [30, 'First names must be no longer than 30 characters.'],
+		required: true
 	},
 	lastName: {
 		type: String,
 		trim: true,
 		minlength: [2, 'Last names must be at least 2 characters.'],
-		maxlength: [30, 'Last names must be no longer than 30 characters.']
+		maxlength: [30, 'Last names must be no longer than 30 characters.'],
```

Comment:
This might be a tad short for names from other countries. Consider bumping it up to at least 64 characters.